client/src/components/Workflow/Invocation/Export/ActionButton.vue

```diff
@@ -0,0 +1,22 @@
+<script setup>
+import { InvocationExportPluginAction } from "./model";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+
+import { ref } from "vue";
+
+const modal = ref(null);
+
+const props = defineProps({
+    action: { type: InvocationExportPluginAction, required: true },
+});
+</script>
+
+<template>
+    <b-button v-b-tooltip.hover.bottom :title="props.action.title" @click="props.action.run(modal)">
+        <font-awesome-icon v-if="props.action.icon" :icon="props.action.icon" />
```

Comment:
```suggestion
        <icon v-if="props.action.icon" :icon="props.action.icon" />
```
The `icon` component is registered globally for font awesome icons and it can be replaced here and in other files too :)

Reply:
I'm torn on `icon` and I don't think we need to use it across the board.  It's okay to be explicit and use the font-awesome-icon directly.

Reply:
Ok, leaving it as `font-awesome-icon` then!